lib/content-services/src/lib/search/components/search-facet-field/search-facet-field.component.html

```diff
@@ -8,8 +8,9 @@
                 <input matInput placeholder="{{ 'SEARCH.FILTER.ACTIONS.SEARCH' | translate }}"
                        [attr.data-automation-id]="'facet-result-filter-'+field.label" [(ngModel)]="field.buckets.filterText">
                 <button *ngIf="field.buckets.filterText" mat-button matSuffix mat-icon-button
+                        [title]="'SEARCH.FILTER.BUTTONS.CLEAR' | translate"
                         (click)="field.buckets.filterText = ''">
-                    <mat-icon role="button" [attr.aria-label]="'SEARCH.FILTER.BUTTONS.CLOSE' | translate">close</mat-icon>
```

Comment:
Why do we want to remove label from the icon?

Reply:
Yeah, right, its a label, keeping it would be better. Thanks for the input.